channels/google/accounts/account-setup.rst

```diff
@@ -0,0 +1,123 @@
+Account Setup
+=============
+
+After you’ve :doc:`registered Zammad as an OAuth app <register-app>`
+in your Google Developer settings,
+you can begin connecting Gmail accounts to Zammad.
+
+☠️ **But first, a word of warning!** The import process does things you might not expect:
+
+.. danger:: 🚯 **Zammad will delete all emails in your inbox
+   during the import process.**
+
+   Use the *Keep Messages on Server* setting to
+   :ref:`disable this behavior <google-keep-messages-on-server>`.
+
+.. warning:: 📮 **Zammad will send an auto-reply message
+   to every email it imports.** (Including the old ones!)
+
+   Make sure to :doc:`disable this behavior </manage/trigger>`
+   prior adding an email account,
+   and to turn it back on once all your messages have been imported.
+
```

Comment:
I also removed the warning about how changing your Google password will invalidate your OAuth tokens. There is already so much information in that section, and I couldn't find a good, logical place to put it, so I decided it wasn't important enough to shoehorn into there.

If you think it's really important, I'm happy to put it back.